pkg/network/vmicontroller/status_test.go

```diff
@@ -90,13 +90,15 @@ var _ = Describe("Status Update", func() {
 		customIfaceName = "custom-iface"
 	)
 
+	clusterConfig := stubClusterConfigChecker{dynamicPodInterfaceNamingEnabled: true}
```

Comment:
Generally I would use a config with default as `off` and then use the other one only for test cases introduced in #13018

Reply:
I have adapt the tests to use off and on where needed.